### ISSUE TITLE
Not able to clean without water (Roborock S7 MaxV Ultra)

### ISSUE THREAD

**Issue by christoph-pieper** [marcelrv/XiaomiRobotVacuumProtocol#85]
Hi!
I'm trying to control my Roborock S7 MaxV Ultra over Openhab. To only vaccum (and not mob) I use the **Roborock_S7_Water_Box_Mode** command (setting it to 200) (see https://github.com/marcelrv/XiaomiRobotVacuumProtocol/blob/master/water_box_custom_mode.md#modes). The Xiaomi App reflects the change.

But when I start room cleaning with **Roborock_S7_Vacuum_Room**, it resets to (vaccum + mob). How can I just use vaccum mode (and not mob mode)?

**Comment by da-mkay** [comment]
Did you find a solution?

I stumbled upon the same issue.
However, I noticed that when I send an `app_stop`, before setting the water box mode followed by an `app_segment_clean`, it seems to respect the water box mode.
Looks promising, but I need to do some more testing.

**Comment by christoph-pieper** [comment]
> Did you find a solution?

No. Im sorry. I didn't continue with my work. To be honest, I was assuming that someone else is much better in implementing / testing than me.

But I would love to continue. I will test your theory :-)
